### PR TITLE
cmd/atlas: support multi-env logging

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

**Review by rotemtam — APPROVED** [review]
💯

File: cmd/atlas/internal/migrate/report.go [hunk 1]
```diff
@@ -476,6 +482,20 @@ func jsonFn(v any, args ...string) (string, error) {
 	return string(b), err
 }
 
+func jsonMerge(objects ...string) (string, error) {
+	var r map[string]any
+	for i := range objects {
+		if err := json.Unmarshal([]byte(objects[i]), &r); err != nil {
```
Comment (giautm) [inline cmd/atlas/internal/migrate/report.go:488]: Interesting, I don't know that we can merge the JSON like this. 💯

**Review by giautm — COMMENTED** [review]
💯

File: cmd/atlas/internal/migrate/report.go (hunk 1 above)
Comment (a8m) [reply cmd/atlas/internal/migrate/report.go:488]: ❤️

**Pull request merged**